pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["poetry==1.3.2"]
+requires = ["poetry==1.8.3"]
```

Comment:
Side comment: I'm forced to override this in the RPM packaging since the chances of having exactly this version of Poetry are not high.

Reply:
Okay, I will relax to make it >= 1.6.1 or something close to that. There was an issue with PY3.13 and Poetry ~1.3.1 (IIRC).